cmd/bridgec/internal/client.go

```diff
@@ -0,0 +1,119 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/rivine/pkg/api"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// CommandLineClient extend for ERC20 commands
+type CommandLineClient struct {
+	*api.HTTPClient
+
+	RootCmd  *cobra.Command
+	ERC20Cmd *cobra.Command
+}
+
+// NewCommandLineClient creates a new CLI client, which can be run as it is,
+// or be extended/modified to fit your needs.
+func NewCommandLineClient(address, name, userAgent string) (*CommandLineClient, error) {
+	if address == "" {
+		address = "http://localhost:23111"
+	}
+	if name == "" {
+		name = "R?v?ne"
+	}
+	client := new(CommandLineClient)
+	client.HTTPClient = &api.HTTPClient{
+		RootURL:   address,
+		UserAgent: userAgent,
+	}
+
+	client.RootCmd = &cobra.Command{
+		Use:   os.Args[0],
+		Short: fmt.Sprintf("%s Client", strings.Title(name)),
+		Long:  fmt.Sprintf("%s Client", strings.Title(name)),
+		Run: Wrap(func() {
+			fmt.Println("Bride Client v1.0.0")
+
+			fmt.Println()
+			fmt.Printf("Go Version   v%s\r\n", runtime.Version()[2:])
+			fmt.Printf("GOOS         %s\r\n", runtime.GOOS)
+			fmt.Printf("GOARCH       %s\r\n", runtime.GOARCH)
+		}),
+	}
+
+	// create command tree
+	client.RootCmd.AddCommand(&cobra.Command{
+		Use:   "version",
+		Short: "Print version information",
+		Long:  "Print version information.",
+		Run: Wrap(func() {
+			fmt.Println("Bride Client v1.0.0")
+
+			fmt.Println()
+			fmt.Printf("Go Version   v%s\r\n", runtime.Version()[2:])
+			fmt.Printf("GOOS         %s\r\n", runtime.GOOS)
+			fmt.Printf("GOARCH       %s\r\n", runtime.GOARCH)
+		}),
+	})
+	client.RootCmd.AddCommand(&cobra.Command{
+		Use:   "stop",
+		Short: fmt.Sprintf("Stop the %s bridge", name),
+		Long:  fmt.Sprintf("Stop the %s bridge.", name),
+		Run: Wrap(func() {
+			err := client.Post("/bridge/stop", "")
+			if err != nil {
+				cli.Die("Could not stop bridge:", err)
+			}
+			fmt.Printf("bridge stopped.\n")
```

Comment:
```suggestion
			fmt.Println("bridge stopped.")
```

No need to use `Printf`, you are not formatting anything, and you are adding a newspace literal char to break the line, so seems to make more sense to use `Println` instead.